src/command/partial/delete.rs

```diff
@@ -0,0 +1,184 @@
+use crate::client::get_studio_client;
+use crate::command::RoverStdout;
+use anyhow::Result;
+use rover_client::query::partial::delete::{self, DeleteServiceResponse};
+use serde::Serialize;
+use structopt::StructOpt;
+
+#[derive(Debug, Serialize, StructOpt)]
+pub struct Delete {
+    /// Variant of the graph in Apollo Studio
+    #[structopt(long, default_value = "current")]
+    #[serde(skip_serializing)]
+    variant: String,
+
+    /// ID of the graph in Apollo Studio to delete a service from
+    #[structopt(long)]
+    #[serde(skip_serializing)]
+    graph_name: String,
+
+    /// Name of the configuration profile (default: "default")
+    #[structopt(long = "profile", default_value = "default")]
+    #[serde(skip_serializing)]
+    profile_name: String,
+
+    /// Name of the implementing service in the graph to delete
+    #[structopt(long)]
+    #[serde(skip_serializing)]
+    service_name: String,
+
+    /// Skip the delete preview as well as the prompt confirming deletion
+    #[structopt(long)]
+    confirm: bool,
```

Comment:
is this just to ignore composition errors? i think the wording of the help text is slightly confusing

Reply:
It's skips the "Confirm" step, where it asks the user if they'd like to proceed with deletion. That step shows a preview of composition errors that would occur if they confirmed the deletion